src/langchain_google_alloydb_pg/engine.py

```diff
@@ -604,3 +605,268 @@ async def _aload_table_schema(
             )
 
         return metadata.tables[table_name]
+
+    def _get_collection_uuid(
+        self,
+        collection_name: str,
+        pg_collection_table_name: Optional[str] = "langchain_pg_collection",
+    ) -> str:
+        """
+        Get the collection uuid for a collection present in PGVector tables.
+
+        Args:
+            collection_name (str): The name of the collection to get the uuid for.
+            pg_collection_table_name (str): The table name which stores the collection uuid to name mappings.
+                Default: "langchain_pg_collection". Optional.
+
+        Returns:
+            The uuid corresponding to the collection.
+        """
+        try:
+            collection_info = self._fetch(
+                query=f"SELECT name, uuid FROM {pg_collection_table_name} WHERE name = '{collection_name}'",
+            )
+            return collection_info[0].uuid
+        except IndexError as e:
+            raise ValueError(f"Collection: {collection_name}, does not exist.")
+
+    def _is_uuid(self, id: str) -> bool:
+        """Check if a string is of type uuid."""
+        try:
+            uuid.UUID(str(id))
+            return True
+        except:
+            return False
+
+    def _insert_single_batch(
+        self,
+        destination_table: str,
+        metadata_column_names: Optional[List[str]],
+        data: Sequence[RowMapping],
+        use_json_metadata: Optional[bool] = False,
+    ) -> None:
+        """
+        Create a batch insert SQL query to insert multiple rows at once.
+        Looks like
+        INSERT INTO MyTable ( Column1, Column2 )
+        VALUES ( Value1, Value2 ), ( Value1, Value2 ), ...
+
+        Args:
+            destination_table (str): The name of the table to insert the data in.
+            metadata_column_names (str): The metadata columns to be created to keep the data in a row-column format.
+                Optional.
+            data (Sequence[RowMapping]): All the data (to be inserted) belonging to a pgvector collection.
+            use_json_metadata (bool): An option to keep the PGVector metadata as json in the AlloyDB table.
+                Default: False. Optional.
+        """
+        row_number = 0
+        params = {}
+
+        if use_json_metadata:
+            insert_query = f"INSERT INTO {destination_table} (LANGCHAIN_ID, CONTENT, EMBEDDING, LANGCHAIN_METADATA) VALUES"
```

Comment:
We get these default values from the file

Reply:
I see the default values hardcoded here: https://github.com/googleapis/langchain-google-alloydb-pg-python/blob/783dcad96b3cf0ae2ac11af826b71dff67a47f1f/src/langchain_google_alloydb_pg/vectorstore.py#L104

Which file can we extract them from?